pkg/translator/ingress_utils.go

```diff
@@ -26,8 +27,8 @@ const (
 	// The resulting name is of the form "<cluster-name>:<ingress-namespace>:<ingress-name>:<service-name>:<service-port>".
 	edgeLBIngressBackendNameFormatString = "%s" + separator + "%s" + separator + "%s" + separator + "%s" + separator + "%s"
 	// edgeLBIngressFrontendNameFormatString is the format string used to compute the name for an EdgeLB frontend corresponding to a given Ingress resource.
-	// The resulting name is of the form "<cluster-name>:<ingress-namespace>:<ingress-name>".
-	edgeLBIngressFrontendNameFormatString = "%s" + separator + "%s" + separator + "%s"
+	// The resulting name is of the form "<cluster-name>:<ingress-namespace>:<ingress-name>:<protocol>".
+	edgeLBIngressFrontendNameFormatString = "%s" + separator + "%s" + separator + "%s" + separator + "%s"
```

Comment:
Wow I hate the use of `separator` here - makes the code harder to read for no reason IMO.

Reply:
Since you hate it so much I'll change the original code.